client/lib/products-values/translations.js

```diff
@@ -114,16 +114,16 @@ export const getJetpackProductsDescriptions = () => {
 	);
 	return {
 		[ CONSTANTS.PRODUCT_JETPACK_BACKUP_DAILY ]: translate(
-			'Always-on backups ensure you never lose your site. Your changes are saved every day with a 30-day archive.'
+			'Never lose a word, image, page, or time worrying about your site'
```

Comment:
I think we need a `.` at the end of the sentence here, right after `'... about your site'`.

Reply:
Absolutely. I noticed it afterwards and fixed it in https://github.com/Automattic/wp-calypso/pull/45067.